### ISSUE TITLE
'cargo package' error message refers to 'cargo publish'

### ISSUE THREAD

**Issue by Wilfred** [rust-lang/cargo#3061]
```
$ cargo package     
error: 1 dirty files found in the working directory:

Cargo.toml

to publish despite this, pass `--allow-dirty` to `cargo publish`
```

This should be referencing 'cargo package' in the message I think.

**Comment by alexcrichton** [comment]
Oh dear, definitely a bug!

**Comment by alexcrichton** [comment]
Should be an easy one to touch up though, we can probably just drop "to `cargo publish`" here.

**Comment by JIghtuse** [comment]
Fixed by #3068

**Comment by alexcrichton** [comment]
Thanks!
